core/tests/protractor/stateEditor.js

```diff
@@ -134,6 +134,30 @@ describe('State editor', function() {
     users.logout();
   });
 
+  it('should skip the customization modal for interactions having no ' +
```

Comment:
I've added an e2e case for re-customizing an interaction. I don't think an e2e case for deletion is required as I believe we inherently check this feature. For example, in this test I have first set a 'Numeric input' and then changed it to 'Continue' input. What happens in between is that the Numeric input gets deleted followed by selecting the Continue input. So if you see the `openInteraction` method in editor.js, what it does is that, if there is an interaction already present it first deletes it and then sets a new interaction. What do you think about this?